### DISCUSSION TITLE
Evita cargar secciones o widgets si están vacíos

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#28] [Announcements]
# Evita cargar secciones o widgets si están vacíos

Blogger tiene muchas etiquetas, algunas de ellas son visibles obligatoriamente en el código fuente de la página y aquí te enseño a ocultarlos.

https://www.zkreations.com/2023/12/ocultar-secciones-widgets-vacios.html

**Comment by gianles** [comment]
Muy buen aporte Daniel, en blogger se pueden hacer cosas sorprendente.

**Reply by danieIabel** [reply]
Hay una gran variedad de cosas que se pueden hacer, con tan solo saber un poco más y compartiré todo aquello que pueda resultar util. Muchas gracias por pasar y comentar amigo.
